L1Trigger/Phase2L1ParticleFlow/src/L1TCorrelatorLayer1PatternFileWriter.cc

```diff
@@ -0,0 +1,355 @@
+#include "L1Trigger/Phase2L1ParticleFlow/interface/L1TCorrelatorLayer1PatternFileWriter.h"
+#include "FWCore/Utilities/interface/Exception.h"
+#include <iostream>
+
+L1TCorrelatorLayer1PatternFileWriter::L1TCorrelatorLayer1PatternFileWriter(const edm::ParameterSet& iConfig,
+                                                                           const l1ct::Event& eventTemplate)
+    : partition_(parsePartition(iConfig.getParameter<std::string>("partition"))),
+      writeInputs_(iConfig.existsAs<std::string>("inputFileName") &&
+                   !iConfig.getParameter<std::string>("inputFileName").empty()),
+      writeOutputs_(iConfig.existsAs<std::string>("outputFileName") &&
+                    !iConfig.getParameter<std::string>("outputFileName").empty()),
+      outputBoard_(-1),
```

Comment:
Are these "constant parameters" needed in future? Because if not I'm not understanding why to define them as parameters and not as constants (e.g. in the class definition).